tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -4,6 +4,7 @@
 
 @pytest.mark.skip
```

Comment:
You should remove the skips on your integration tests. If you do, you'll see there are some errors. See my comment in `vendor.py` for what's going on there.